kivymd/uix/snackbar.py

```diff
@@ -337,15 +343,11 @@ class BaseSnackbar(MDCard):
 
     Set size_hint_x like below:
 
+    .. code-block:: python
+
     `size_hint_z = (
         Window.width - (snackbar_x * 2)
     ) / Window.width`
```

Comment:
See how to format code blocks correctly - https://github.com/kivymd/KivyMD/blob/master/kivymd/uix/backdrop.py#L34

Reply:
is it right now?

Reply:
I didn't understand your question ...

Reply:
Is the formatting of the code block correct in the PR now? I updated it.